symplyphysics/laws/kinematic/rotational_inertia/geometries/thin_rod_about_axis_through_center_perpendicular_to_length.py

```diff
@@ -0,0 +1,77 @@
+from sympy import Eq, Symbol as SymSymbol, solve, integrate
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+)
+from symplyphysics.core.expr_comparisons import expr_equals
+from symplyphysics.laws.kinematic.rotational_inertia import (
+    rotational_inertia_of_particle as rotational_inertia_def,
+)
+
+# Description
+## A thin rod is rotating about an axis that passes through its center perpendicular to its length.
+
+# Law: I = 1/12 * M * L**2
+## I - rotational inertia of thin rod
+## M - mass of rod
+## L - length of rod
+
+rotational_inertia = Symbol("rotational_inertia", units.mass * units.length**2)
+mass = Symbol("mass", units.mass)
+length = Symbol("length", units.length)
+
+law = Eq(rotational_inertia, mass * length**2 / 12)
+
+
+# Derive law from definition of rotational inertia
```

Comment:
Why not take the inertia of a slab and assume its width is zero? I think it is equivalent to a thin rod, and even to a more general case of a rotating plane.

Reply:
My goal was mostly to show how to use the most basic definition to find the integral rotational inertia, which might be useful more more complicated cases. But I agree, it's a subcase of a slab with zero width.

Reply:
It's hard to expect someone would  find this law, when trying to find out how to integrate over rotating body. Let's use easy way to prove it and use complex way, when it's unavoidable, eg some weird body form.